osd_getting_started/osd-getting-started.adoc

```diff
@@ -12,7 +12,7 @@ Follow this getting started document to quickly create a {product-title} cluster
 [id="osd-getting-started-prerequisites"]
 == Prerequisites
 
-* You reviewed the xref:../osd_architecture/osd-understanding.adoc#osd-understanding[introduction to {product-title}] and the documentation on xref:../osd_architecture/osd-architecture.adoc#osd-architecture[architecture concepts].
+* You reviewed the xref:../osd_architecture/osd-understanding.adoc#osd-understanding[introduction to {product-title}] and the documentation on xref:../architecture/index.adoc[architecture concepts].
```

Comment:
All xrefs need an anchor ID even if they link to the top of the assembly, [per our guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references):
```suggestion
* You reviewed the xref:../osd_architecture/osd-understanding.adoc#osd-understanding[introduction to {product-title}] and the documentation on xref:../architecture/index.adoc#architecture-overview[architecture concepts].
```